README.md

```diff
@@ -0,0 +1,46 @@
+# Guardian Commercial Templates
+
+This projects creates bespoke ads that fit perfectly on [The Guardian][],
+known as “native” in Google Ad Manager parlance.
+
+[the guardian]: https://theguardian.com/
+
+Aiming to provide a great developer and reader experience, this project uses
+[Svelte][] to build the template, and [SvelteKit][] to help preview development.
+You can see the current templates on [guardian.github.io/commercial-templates](https://guardian.github.io/commercial-templates/).
+
+[svelte]: https://svelte.dev/
+[sveltekit]: https://kit.svelte.dev/
+
+This project is a rewrite and work-in-progress. Legacy templates can still be
+found in the [`/legacy` folder](/legacy)
```

Comment:
Is this enough of an intro?

Reply:
Seems like more than enough to get started with this repo